frontend/packages/console-shared/locales/en/activity-card.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This feels like it might be a little too granular. Should there be a cluster-overview or dashboard namespace for all of these strings?

The browser will have to make a separate XHR for all of these files, which has a performance impact.